refreeze.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```

Comment:
Why do we also have `refreeze.sh`

Reply:
Because people were complaining that it was annoying to refreeze the packages so I wrote a script to take care of it